vaporetto/src/sentence.rs

```diff
@@ -1089,6 +1095,13 @@ impl<'a, 'b> Sentence<'a, 'b> {
         &mut self.tags
     }
 
+    /// Returns a slice of tag scores.
```

Comment:
```suggestion
    /// Returns a slice of tag scores.
    ///
    /// The returned slice has the same length as the sentence, and
    /// the `i`-th element corresponds to the `i`-th boundary.
    /// If and only if the `i`-th is a word boundary, it is a `Some` element.
    ///
    /// Each element is a pair as follows:
    ///  * 1st - Tag-string candidates for each tag class.
    ///  * 2nd - Prediction scores for each candidate.
    ///
    /// Let us denote the 1st one by `X[i][j]` and the 2nd one by `Y[k]`.
    /// `Y[k]` stores the score of the `k`-th candidate encountered when
    /// scanning `X` in row-major order, skipping tag classes such that
    /// `X[i]` is of length one.
```
The returned value is very complicated and hard to understand. I think it would be better to define a struct that explains the returned value; however, if you consider this function to be minor and such a struct definition is less important, I will respect that.

In any case, to avoid the cost of understanding this function, its specifications would have to be clarified. Could you clarify the specifications? (My suggestion is one example based on my understanding.)